src/common/src/system_param/reader.rs

```diff
@@ -77,7 +77,15 @@ impl SystemParamsReader {
     }
 
     pub fn pause_on_next_bootstrap(&self) -> bool {
-        self.prost.pause_on_next_bootstrap.unwrap_or(false)
+        self.prost
+            .pause_on_next_bootstrap
+            .unwrap_or_else(|| default::pause_on_next_bootstrap().unwrap())
+    }
+
+    pub fn enable_tracing(&self) -> bool {
+        self.prost
+            .enable_tracing
+            .unwrap_or_else(|| default::enable_tracing().unwrap())
```

Comment:
```suggestion
            .unwrap_or_else(|| default::pause_on_next_bootstrap().unwrap_or_default())
    }

    pub fn enable_tracing(&self) -> bool {
        self.prost
            .enable_tracing
            .unwrap_or_else(|| default::enable_tracing().unwrap_or_default())
```
?

Reply:
They're always `Some` according to system-param definitions.

Reply:
Since they're generated, we may also add a new function returning the unwrapped type.